ext/openssl/openssl.c

```diff
@@ -1178,6 +1178,12 @@ PHP_MINIT_FUNCTION(openssl)
 #ifdef X509_PURPOSE_ANY
 	REGISTER_LONG_CONSTANT("X509_PURPOSE_ANY", X509_PURPOSE_ANY, CONST_CS|CONST_PERSISTENT);
 #endif
+#ifdef X509_PURPOSE_OCSP_HELPER
```

Comment:
I don't think we need ifdefs here as all supported versions (1.0.1+) have got it defined. When you are in it, you could also remove ifdef for `X509_PURPOSE_ANY`.